src/packages/website/templates/website/bower.json

```diff
@@ -24,6 +25,7 @@
     }
   },
   "resolutions": {
-    "angular": "1.5.x"
+    "angular": "1.5.x",
+    "angular-animate": "1.5.9"
```

Comment:
it worth to fix either both `angular` and `angular-animate` at 1.5.x or at 1.5.9, otherwise you'll have the same problem with angular 1.5.10.
The same applies to the line above.